includes/services/OpenLayers/jquery.openlayers.js

```diff
@@ -92,8 +94,8 @@
 				}
 
 				if (bounds != null) bounds.extend(location.lonlat); // Extend the bounds when no center is set.
-				var marker = this.getOLMarker(curLayer, location);
-				this.markers.push({
+				var marker = _this.getOLMarker(curLayer, location);
+				_this.markers.push({
```

Comment:
It is defined a few lines down, but I reverted the change because it's unnecessary.